cpp/proto/serializer.cc

```diff
@@ -298,63 +299,25 @@ SerializeResult Serializer::SerializeSCTList(
 // static
 SerializeResult Serializer::SerializeDigitallySigned(
     const DigitallySigned& sig, string* result) {
+  result->clear();
   return WriteDigitallySigned(sig, result);
 }
 
-size_t SerializedListLength(const repeated_string& in, size_t max_elem_length,
-                            size_t max_total_length) {
-  size_t elem_prefix_length = PrefixLength(max_elem_length);
-  size_t total_length = 0;
-
-  for (int i = 0; i < in.size(); ++i) {
-    if (in.Get(i).size() > max_elem_length ||
-        max_total_length - total_length < elem_prefix_length ||
-        max_total_length - total_length - elem_prefix_length <
-            in.Get(i).size())
-      return 0;
-
-    total_length += elem_prefix_length + in.Get(i).size();
-  }
-
-  return total_length + PrefixLength(max_total_length);
-}
-
-
 // static
 SerializeResult Serializer::SerializeList(const repeated_string& in,
                                           size_t max_elem_length,
                                           size_t max_total_length,
                                           string* result) {
   std::string output;
   SerializeResult res =
-      ::WriteList(in, max_elem_length, max_total_length, &output);
+      cert_trans::serialization::WriteList(in, max_elem_length,
+                                           max_total_length, &output);
```

Comment:
It's fine if you don't (just one use that I saw in this file), but could add a `using` for it at the top to make this line a bit nicer.